### PR TITLE
[SCM-936] extract common code

### PR DESCRIPTION
**Pull request by elharo**
@michael-o Let's see if the test still flakes.

### CODE REVIEW

File: maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java [hunk 1]
```diff
@@ -63,32 +62,29 @@ protected void setUp()
...
+        String connectionUrl = mojo.getConnectionUrl();
+        connectionUrl = connectionUrl.replace( "${basedir}", getBasedir() );
+        connectionUrl = connectionUrl.replace( '\\', '/' );
+        mojo.setConnectionUrl( connectionUrl );
+    }
+
     public void testTag()
         throws Exception
     {
-        if ( !ScmTestCase.isSystemCmd( SvnScmTestUtils.SVNADMIN_COMMAND_LINE ) )
```
Comment (michael-o) [inline maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java:79]: Why did you remove this test?

Comment (elharo) [reply maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java:79]: It's not a test, it's an assumption and it's duplicative of the same code in the setUp method. If the code fails there, then the test isn't set up correctly here.

Comment (michael-o) [reply maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java:79]: Is the test really skipped w/o it?

Comment (elharo) [reply maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java:79]: OK. I've added an explicit assumption in the setUp method to cover this.

Comment (elharo) [reply maven-scm-plugin/src/test/java/org/apache/maven/scm/plugin/TagMojoTest.java:79]: Jenkins passed

**Comment by jira-importer** [comment]
Resolve #1162
